src/modules/UI/scenes/CreateWallet/CreateWalletSelectCrypto.ui.js

```diff
@@ -0,0 +1,186 @@
+// @flow
+
+import React, {Component} from 'react'
+import {Actions} from 'react-native-router-flux'
+import {
+  ActivityIndicator,
+  Alert,
+  View,
+  Keyboard,
+  TouchableHighlight,
+  Image
+} from 'react-native'
+import Text from '../../components/FormattedText'
+import {SecondaryButton, PrimaryButton} from '../../components/Buttons'
+import {FormField} from '../../../../components/FormField.js'
+import SearchResults from '../../components/SearchResults'
+
+import styles, {styles as stylesRaw} from './style.js'
+import s from '../../../../locales/strings.js'
+import PLATFORM from '../../../../theme/variables/platform'
+import Gradient from '../../components/Gradient/Gradient.ui'
+import * as UTILS from '../../../utils'
+import type {GuiWalletType, FlatListItem, DeviceDimensions} from '../../../../types'
+
+const WALLET_TYPE_PICKER_PLACEHOLDER = s.strings.create_wallet_choose_crypto
+
+const BACK_TEXT = s.strings.title_back
+const INVALID_INPUT_TITLE = s.strings.create_wallet_invalid_input
+const INVALID_DATA_TEXT = s.strings.create_wallet_select_valid_crypto
+const NEXT_TEXT = s.strings.string_next_capitalized
```

Comment:
Do not add extra indirection for variables. Does not add any value and it more inconvenient to change.